src/render-html/js.js

```diff
@@ -10,6 +10,8 @@ const sendMessage = msg => {
 const isNearByPositions = (x1, y1, x2, y2) =>
   x1 && y1 && x2 && y2 && Math.abs(x1 - x2) < 10 && Math.abs(y1 - y2) < 10;
 
+const validateTouchAreaForLongPress = (x1, y1) => x1 < 20;
```

Comment:
That looks like a random value. Do you think it is robust and good way to differentiate drawer sliding? I doubt it.